SELF_HOSTED_INSTRUCTIONS.md

```diff
@@ -0,0 +1,70 @@
+This example shows a basic approach at deploying a working Uwazi.
+
+## First one-time setup
+
+Ensure you have all [Uwazi dependencies](https://github.com/huridocs/uwazi#dependencies) correctly configured an running.
+
+During the first time installation, create a basic folder structure with folders for potential uploaded files, files created by Uwazi and the production-build code for Uwazi.
+To create this folder structure run:
+
+```
+mkdir -p uwazi/log uwazi/uploaded_documents uwazi/custom_uploads uwazi/temporal_files
+.
+├── /home/user/uwazi
+│   ├── log/
+│   ├── uploaded_documents/
+│   ├── custom_uploads/
+│   ├── temporal_files/
+```
+
+- From within the `uwazi` folder, download the latest release .tar file:
+
+`wget https://github.com/huridocs/uwazi/releases/latest/download/uwazi.tgz`
+
+- Unpack the file. This will create a `prod` folder with the latest Uwazi production code:
+
+`tar xvf uwazi.tgz`
+
+- Rename to the `prod` folder to `lastest` and delete the .tar file:
+
+`mv prod latest && rm uwazi.tgz`
+
+- you should have now
+
+```
+.
+├── uwazi
+│   ├── latest/
+│   ├── log/
+│   ├── uploaded_documents/
+│   ├── custom_uploads/
+│   ├── temporal_files/
+```
+
+- From within `uwazi/latest`, create a blank state with the desired database name. The platform deafults to `uwazi_development` if you don't define any name.
+
+`NODE_ENV=production yarn blank-state my_db_name`
+
+## Update Uwazi to a new release
+
+- Rename or delete previous `latest` folder
+- From within `uwazi` folder, download the latest release .tar file, unpack, rename to `latest` and remove the previou .tar file.
+
+```
+wget https://github.com/huridocs/uwazi/releases/latest/download/uwazi.tgz
+tar xvf uwazi.tgz
+mv prod latest && rm uwazi.tgz
+```
+
+- From within `uwazi/latest` run any migrations needed since your previous version and reindex the Elastic Search index:
+
+`NODE_ENV=production yarn migrate-and-reindex`
```

Comment:
This will reindex `uwazi_development` and not include the file paths, so there needs to be some mention of the env variables here and maybe use the same names used in the examples presented here..

Reply:
fixed :+1: